CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ section for the next release.
 
 For more information about this file see also [Keep a Changelog](http://keepachangelog.com/) .
 
-## [1.8.0] - 2024-07-12
+## [Unreleased]
```

Comment:
Fixing a merge error -- released material starts on line 23 and has the right heading already